### PR TITLE
[chore] move cassandra exporter to generated lifecycle tests

### PR DESCRIPTION
**Pull request by Frapschen**
Relates to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27849

duo to `cluster.CreateSession()` func, event if set `skip_lifecycle: true` the Lifecycle test still fail:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/1de0e36f76a5e6a9a9ced094b7b90529ea3beae6/exporter/cassandraexporter/exporter_logs.go#L26-L28
test log:
```
=== RUN   TestComponentLifecycle
=== RUN   TestComponentLifecycle/logs-shutdown
    generated_component_test.go:75: 
        	Error Trace:	/Users/fraps/daocloud/github-code/opentelemetry-collector-contrib/exporter/cassandraexporter/generated_component_test.go:75
        	Error:      	Received unexpected error:
        	            	cannot configure cassandra traces exporter: gocql: unable to create session: unable to discover protocol version: dial tcp 127.0.0.1:9042: connect: connection refused
        	Test:       	TestComponentLifecycle/logs-shutdown
=== RUN   TestComponentLifecycle/logs-lifecycle
=== RUN   TestComponentLifecycle/traces-shutdown
    generated_component_test.go:75: 
        	Error Trace:	/Users/fraps/daocloud/github-code/opentelemetry-collector-contrib/exporter/cassandraexporter/generated_component_test.go:75
        	Error:      	Received unexpected error:
        	            	cannot configure cassandra traces exporter: gocql: unable to create session: unable to discover protocol version: dial tcp 127.0.0.1:9042: connect: connection refused
        	Test:       	TestComponentLifecycle/traces-shutdown
```

we need more discuss for it @atoulme

### CODE REVIEW

**Comment by atoulme** [comment]
Please open a bug to follow up, the component should not fail on creation, but on start. This is not a fatal error either; the collector should keep trying to connect.

**Review by atoulme — APPROVED** [review]
LGTM (I made those changes)

File: exporter/cassandraexporter/metadata.yaml [hunk 1]
```diff
@@ -8,3 +8,6 @@ status:
   codeowners:
     active: [atoulme, emreyalvac]
 
+tests:
+  config:
+  skip_lifecycle: true
```
Comment (TylerHelmuth) [inline exporter/cassandraexporter/metadata.yaml:13]: Should they still be skipped or does this PR make it so this can be enabled?

Comment (atoulme) [reply exporter/cassandraexporter/metadata.yaml:13]: this PR doesn't allow them to be enabled yet.

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**